Automattic/wp-calypso
# Add search spotlight placeholder to JITMs

Related to https://github.com/Automattic/dotcom-forge/issues/2743

## Proposed Changes

In #78010 we've passed the search parameters to the WPCOM call, and the JITMs were only returned from the backend. This PR adds a placeholder message to the banner on top of the search results, to avoid layout shifts when the JITM gets rendered. We've tried a similar approach with a default message being returned from the backend, but that has resulted in a noticeable delay. Related thread: pdh6GB-2Ty-p2#comment-3665

## Testing Instructions



* Patch changes in D114133-code and sandbox `public-api.wordpress.com`
* With a simple site, go to `/plugins`
* Visit any category/search for any search term that is matched in the diff
* Verify that before the JITM is loaded from WPCOM, the loading placeholder is being displayed

Video of the banner loading:

https://github.com/Automattic/wp-calypso/assets/11555574/017c7ba2-bf44-4701-804e-3d3124456be7

## Pre-merge Checklist
